corehq/apps/hqadmin/tasks.py

```diff
@@ -198,6 +198,6 @@ def track_pg_limits():
             cursor.execute(query)
             results = cursor.fetchall()
             for table, sequence in results:
-                cursor.execute(f'select last_value from {sequence}')
+                cursor.execute(f'select last_value from "{sequence}"')
```

Comment:
There's no way to get execute to quote this for us, right? i.e. it only does that for values and not table names, right? I feel funny about plopping strings into SQL but since we're getting this from a real existing sequence name is there no risk? (I don't know, like can a sequence name include a `"` character for example?)

Reply:
Yeah, since we are getting it from sql, the name has to be a valid postgres identifier, and it is my understanding that those are always quotable. Specifically they are limited to letters, numbers, underscores, or dollar signs https://www.postgresql.org/docs/9.6/sql-syntax-lexical.html

Reply:
Great, thanks

Reply:
If you really want you can use the psycopg2 library to generate the SQL:
```suggestion
                cursor.execute(sql.SQL('select last_value from {}').format(sql.Identifier(sequence)))
```

Reply:
Neat, I didn't know about that